test.go

```diff
@@ -203,6 +207,18 @@ func (t *Test) SaveLatestCSV() {
 	}
 }
 
+// StoreLatestToPrometheus stores latest version results to prometheus pushgateway
+func (t *Test) StoreLatestToPrometheus() {
```

Comment:
If promConfig and ciConfig are only used here, I would consider not adding them to the Test struct and instead pass them as parameters to this function

Reply:
Done